unumpy/tests/test_numpy.py

```diff
@@ -291,6 +291,13 @@ def test_multiple_output(backend, method, args, kwargs):
         (np.ones_like, ([1, 2, 3],), {}),
         (np.zeros, ((1, 2, 3),), {}),
         (np.zeros_like, ([1, 2, 3],), {}),
+        (np.asanyarray, ([1, 2, 3],), {}),
+        (np.ascontiguousarray, ([1, 2, 3],), {}),
+        (np.copy, ([1, 2, 3],), {}),
+        (np.frombuffer, (b"123",), {}),
```

Comment:
This takes the raw ascii codes for "123" and tries to interpret them as various different numeric types. Instead you would be better off writing a dedicated test for `frombuffer` that uses the output from original numpy's `tobytes`:
```python
>>> dtype = np.int16
>>> onp.array([1, 2, 3], dtype=dtype).tobytes()
b'\x01\x00\x02\x00\x03\x00'
>>> onp.frombuffer(b'\x01\x00\x02\x00\x03\x00', dtype=dtype)
array([1, 2, 3], dtype=int16)
```